src/vpoller/vsphere/tasks.py

```diff
@@ -31,19 +31,7 @@
     - https://www.vmware.com/support/developer/vc-sdk/
 
 """
-
-
-import ssl
-
-try:
-    _create_unverified_https_context = ssl._create_unverified_context
-except AttributeError:
-    # Legacy Python that doesn't verify HTTPS certificates by default
-    pass
-else:
-    # Handle target environment that doesn't support HTTPS verification
-    ssl._create_default_https_context = _create_unverified_https_context
-
```

Comment:
This one here is incorrect as it will introduce back issue #113 

Please rebase your master branch first against the vpoller/master branch.

Thanks!